src/components/TodoForm/TodoForm.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const TodoForm = (
+  {
+    usersList,
+    changeTask,
+    changeUserId,
+    value,
+    id,
+    addNewTask,
+    inputValue,
+    personError,
+    titleError,
+  },
+) => (
+  <form onSubmit={(e) => {
+    e.preventDefault();
+  }}
+  >
+    <label htmlFor="title" className="form__title">
+      <input
+        type="text"
+        id="title"
+        onChange={changeTask}
+        value={inputValue}
+      />
+    </label>
+    {titleError || <span> Please enter the title </span>}
+    <br />
+    <select
+      onChange={changeUserId}
+      value={value}
+      className="form__person"
+    >
+      <option value="0">Choose person</option>
+      {usersList.map(item => (
+        <option value={item.id} key={item.id}>{item.name}</option>
+      ))}
+    </select>
+    {personError || <span>Please choose a user</span>}
+    <br />
+    <button
+      onClick={() => addNewTask(id)}
+      className="form__btn"
+      type="submit"
+    >
+      add task
+    </button>
+  </form>
+);
+
+TodoForm.propTypes = {
+  usersList: PropTypes.arrayOf(PropTypes.shape({
+    id: PropTypes.number.isRequired,
+    name: PropTypes.string.isRequired,
+    username: PropTypes.string.isRequired,
+    email: PropTypes.string.isRequired,
+    address: PropTypes.object.isRequired,
+
```

Comment:
propTypes is what you use in the component, if you don't use some parts of the object then there is no need to write them in the propTypes